packages/@n8n/api-types/src/dto/oidc/config.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from 'zod';
+import { Z } from 'zod-class';
+
+export class OidcConfigDto extends Z.class({
+	clientId: z.string().min(1),
+	clientSecret: z.string().min(1),
+	discoveryEndpoint: z.string().url(),
+	loginEnabled: z.boolean().optional().default(false),
```

Comment:
I'm not sure I see why is loginEnabled part of the DTO. It was my understanding that whether oidc is enabled or not came from the global config (environment variable)

Reply:
This is just used for turning OIDC on and off via the FE. It is similar to the SAML toggle dto

Reply:
ah okay :+1: